genbindings.sh

```diff
@@ -171,6 +175,9 @@ else
 	echo "WARNING: Can't use address sanitizer on non-Linux, non-OSX non-x86 platforms"
 fi
 
+cargo rustc -v --target=wasm32-wasi -- -C embed-bitcode=yes || echo "WARNING: Failed to generate WASM LLVM-bitcode-embedded library"
```

Comment:
I had to run `rustup target add wasm32-wasi` to get these lines to work. It could be worth adding a reeeaally simple `README` to `c-bindings-gen` just saying that latest `cbindgen` and this additional target is needed (unless that already exists somewhere that I'm forgetting). Fine to leave that for follow-up though.